p2panda-rs/src/storage_provider/traits/entry_store.rs

```diff
@@ -8,11 +8,13 @@ use crate::identity::Author;
 use crate::schema::SchemaId;
 use crate::storage_provider::errors::EntryStorageError;
 use crate::storage_provider::traits::AsStorageEntry;
+use mockall::automock;
```

Comment:
I've been planning to make a simple style guide for our codebase but I haven't done it yet :smile_cat: 

We're fairly strict with how we organize / group imports:

- `std` level imports
- external crate imports
- `crate` level imports
- `super` level imports (only in tests)